Makefile

```diff
@@ -54,10 +70,10 @@ else
 endif
 
 quicktest:
-	$(PATH_WITH_HELM) go test -short ./ $(addprefix ./,$(GO_PACKAGES))
+	${DEV_ENV_CMD} go test -short ./ $(addprefix ./,$(GO_PACKAGES))
 
-test: test-style
-	$(PATH_WITH_HELM) go test -v ./ $(addprefix ./,$(GO_PACKAGES))
+test:
+	${DEV_ENV_CMD} go test -v ./ $(addprefix ./,$(GO_PACKAGES))
 
 test-style:
```

Comment:
This target has steps that can be containerized.  We shouldn't assume contributors have `go`, `gofmt`, and `golint` installed (and even if they do, we should not assume they have correct versions of each).